libsplinter/src/admin/service/shared.rs

```diff
@@ -664,6 +664,38 @@ impl AdminServiceShared {
                 self.propose_circuit(payload, "local".to_string())
             }
             CircuitManagementPayload_Action::CIRCUIT_PROPOSAL_VOTE => {
+                let proposal_vote = payload.get_circuit_proposal_vote();
```

Comment:
Can you add a log statement somewhere so we can see why the vote failed, I don't see any indication in the logs (maybe we should add one for circuit proposal too?)

Reply:
Added as a new commit (at the submit handler layer).